src/main/resources/fileTemplates/code/project_settings.gradle.kts.ft

```diff
@@ -1,12 +1,48 @@
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
 pluginManagement {
     repositories {
-        google()
         gradlePluginPortal()
+        maven(url = "https://plugins.gradle.org/m2/")
+        google{
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+            }
+            mavenContent{
+                includeGroupAndSubgroups("androidx")
+                includeGroupAndSubgroups("com.android")
+                includeGroupAndSubgroups("com.google")
+            }
+        }
         mavenCentral()
         maven("https://maven.pkg.jetbrains.space/public/p/compose/dev")
+        maven(url = "https://jitpack.io")
+        maven(url = "https://s3.amazonaws.com/repo.commonsware.com")
+        maven(url = "https://jogamp.org/deployment/maven")
+    }
+    plugins {
+
     }
 }
 
+@Suppress("UnstableApiUsage")
+dependencyResolutionManagement {
+    repositories {
+        google {
+            mavenContent{
+                includeGroupAndSubgroups("androidx")
+                includeGroupAndSubgroups("com.android")
+                includeGroupAndSubgroups("com.google")
+            }
+        }
+        mavenCentral()
+        maven(url = "https://jitpack.io")
+        maven(url = "https://s3.amazonaws.com/repo.commonsware.com")
+        maven(url = "https://maven.pkg.jetbrains.space/public/p/compose/dev")
+        maven(url = "https://jogamp.org/deployment/maven")
```

Comment:
Here too minus the middle one.

Reply:
sorry

I sent the wrong PR

I mainly want to change the versions of various libs (especially the kotlin part, which I want to upgrade to 2.0.20 or above)

I also added my commonly used repos and libs, and then sent the PR without distinguishing them.

Mainly I want to know if this plugin can be modified to support kotlin 2.0 or above and more lib and mvvm templates?